### ISSUE TITLE
implement timeout

### ISSUE THREAD

**Issue by kdabir** [kdabir/dq#11]
no idea how yet, but sbt gets stuck sometimes (may be it was trying to download the internet).

**Comment by dexterous** [comment]
> may be it was trying to download the internet

For the `sbt-version` command?!?

**Comment by kdabir** [comment]
yeah, now don't remember exactly but `sbt` on fresh machine caused this..

**Comment by dexterous** [comment]
Goddammit Scala! Do you _have_ to make everything just that little bit harder!?!
